src/notation/internal/notationinteraction.cpp

```diff
@@ -2012,115 +2116,201 @@ void NotationInteraction::startEditText(Element* element, const PointF& cursorPo
         return;
     }
 
-    m_textEditData.startMove = cursorPos;
+    m_editData.startMove = cursorPos;
 
-    if (isTextEditingStarted()) {
+    if (isTextEditStarted()) {
         // double click on a textBase element that is being edited - select word
-        Ms::TextBase* textBase = Ms::toTextBase(m_textEditData.element);
-        textBase->multiClickSelect(m_textEditData, Ms::MultiClick::Double);
-        textBase->endHexState(m_textEditData);
+        Ms::TextBase* textBase = Ms::toTextBase(m_editData.element);
+        textBase->multiClickSelect(m_editData, Ms::MultiClick::Double);
+        textBase->endHexState(m_editData);
         textBase->setPrimed(false);
     } else {
-        m_textEditData.clearData();
+        m_editData.clearData();
 
-        m_textEditData.element = element;
+        m_editData.element = element;
 
-        if (m_textEditData.element->isTBox()) {
-            m_textEditData.element = toTBox(m_textEditData.element)->text();
+        if (m_editData.element->isTBox()) {
+            m_editData.element = toTBox(m_editData.element)->text();
         }
 
-        element->startEdit(m_textEditData);
+        element->startEdit(m_editData);
     }
 
     notifyAboutTextEditingStarted();
     notifyAboutTextEditingChanged();
 }
 
+static qreal nudgeDistance(const Ms::EditData& editData)
+{
+    qreal spatium = editData.element->spatium();
+    if (editData.element->isBeam()) {
+        if (editData.modifiers & Qt::ControlModifier) {
+            return spatium;
+        } else if (editData.modifiers & Qt::AltModifier) {
+            return 4 * spatium;
+        } else {
+            return 0.25 * spatium;
+        }
+    } else {
+        if (editData.modifiers & Qt::ControlModifier) {
+            return Ms::MScore::nudgeStep10 * spatium;
```

Comment:
For some bizarre reason uncrustify wants it to be `return Ms::MScore::nudgeStep10* spatium;`

Reply:
I ended up swapping those operands just to avoid this...